test/emqx_protocol_SUITE.erl

```diff
@@ -420,5 +667,7 @@ raw_send_serialize(Packet, Opts) ->
     emqx_frame:serialize(Packet, Opts).
 
 raw_recv_parse(P, ProtoVersion) ->
-    emqx_frame:parse(P, {none, #{max_packet_size => ?MAX_PACKET_SIZE,
-                                 version         => ProtoVersion}}).
+    emqx_frame:parse(P,
+		     {none,
+		      #{max_packet_size => ?MAX_PACKET_SIZE,
+			version => ProtoVersion}}).
```

Comment:
Our coding style (maybe different to the common Erlang coding guideline) uses 4 spaces indentation.

Reply:
Revert all your changes regarding to the code format.